src/components/members-list/index.js

```diff
@@ -1,20 +1,40 @@
+/* eslint-disable consistent-return */
+/* eslint-disable array-callback-return */
 import React from 'react';
 import MemberListItem from '@components/member-list-item';
 import classNames from '@components/members-list/member-list.module.scss';
 import { membersContext } from '@store/members/members-context';
 
-const MembersList = () => {
+const MembersList = ({ searchTerm }) => {
   const {
     state: { membersArr },
   } = membersContext();
   if (membersArr) {
     return (
       <div className={classNames.container}>
-        {membersArr.map((ele) => (
-          <React.Fragment key={ele.id}>
-            <MemberListItem memberDetails={ele} />
-          </React.Fragment>
-        ))}
+        {membersArr.length > 0 &&
+          membersArr
+            .filter((ele) => {
+              if (searchTerm !== '') {
+                if (
+                  ele.first_name
+                    .toLowerCase()
+                    .includes(searchTerm.toLocaleLowerCase()) ||
+                  ele.last_name
+                    .toLowerCase()
+                    .includes(searchTerm.toLocaleLowerCase())
+                ) {
+                  return ele;
+                }
+              } else {
+                return ele;
+              }
+            })
+            .map((ele) => (
+              <React.Fragment key={ele.id}>
+                <MemberListItem memberDetails={ele} />
```

Comment:
Since there is only one component to render any specific reason to use fragments?